### PR TITLE
Avatar an dbadge

### PR DESCRIPTION
**Pull request by prakash855**

### CODE REVIEW

**Comment by Kevin-Solomon** [comment]
It would be nice if there was a description to the PR.

**Comment by prakash855** [comment]
> It would be nice if there was a description to the PR.

File: components/Avatar/Avatar.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                <!-- image avatars -->
+                <div class="content_subheading_heading">Image Avatars</div>
+                <p class="content_text">Image avatars can be created by passing <span class="highlights">img</span>
+                    props as <span class="highlights">src</span> to the component.</p>
+                <div class="component_container flex shadow wrap">
+                    <img class="avatar_xl round responsive_image" src="../../assets/avatar.jpg" alt="">
+                    <img class="avatar_lg round responsive_image" src="../../assets/avatar.jpg" alt="">
+                    <img class="avatar_md round responsive_image" src="../../assets/avatar.jpg" alt="">
+                    <img class="avatar_sm round responsive_image" src="../../assets/avatar.jpg" alt="">
+                    <img class="avatar_xs round responsive_image" src="../../assets/avatar.jpg" alt="">
```
Comment (kart-c) [inline components/Avatar/Avatar.html:73]: `alt` attributes should be filled here

**Review by kart-c — APPROVED** [review]
The landing page animation looks good, Prakash. I would suggest to add the `alt` attributes to all `img` tags, as it helps in accessibility

File: components/Avatar/Avatar.html (hunk 1 above)
Comment (prakash855) [reply components/Avatar/Avatar.html:73]: noted

**Review by AtulPant2704 — CHANGES_REQUESTED** [review]
You have so many unwanted files in this PR, test.js & settings.json is not required and there are many empty files as well, remove all those files. 
And add description in your PR and a codeandbox live link too so that it is easier for the reviewer to review.

**Comment by prakash855** [comment]
> You have so many unwanted files in this PR, test.js & settings.json is not required and there are many empty files as well, remove all those files. And add description in your PR and a codeandbox live link too so that it is easier for the reviewer to review.

Noted!
files like index.js do have some functionalities added into it

**Review by palash23jain — APPROVED** [review]

**Pull request merged**